.github/workflows/cibuildwheels.yml

```diff
@@ -138,17 +139,17 @@ jobs:
           python -m pytest
 
 
-  upload_pypi:
-    needs: [ build_wheels, build_sdist ]  # last but not least
-    runs-on: ubuntu-latest
-    if: startsWith(github.event.ref, 'refs/tags')
-    steps:
-      - uses: actions/download-artifact@v2
-        with:
-          name: artifact
-          path: dist
-
-      - uses: pypa/gh-action-pypi-publish@master
-        with:
-          user: __token__
-          password: ${{ secrets.blosc_pypi_secret }}
+#  upload_pypi:
+#    needs: [ build_wheels, build_sdist ]  # last but not least
+#    runs-on: ubuntu-latest
+#    if: startsWith(github.event.ref, 'refs/tags')
+#    steps:
+#      - uses: actions/download-artifact@v2
+#        with:
+#          name: artifact
+#          path: dist
+#
+#      - uses: pypa/gh-action-pypi-publish@master
+#        with:
+#          user: __token__
+#          password: ${{ secrets.blosc_pypi_secret }}
```

Comment:
This needs to be uncommented

Reply:
I am waiting to merge https://github.com/Blosc/c-blosc2/pull/318 to uncomment it because of the renamed library files